packages/databricks-vscode/src/configuration/ConnectionManager.ts

```diff
@@ -275,25 +273,11 @@ export class ConnectionManager implements Disposable {
                 `Can't connect to the workspace`,
                 e
             );
-            if (e instanceof ProcessError) {
-                window
-                    .showErrorMessage(
-                        `Can't connect to the workspace. Error executing Databricks CLI command.`,
-                        "Show Logs"
-                    )
-                    .then((choice) => {
-                        if (choice === "Show Logs") {
-                            commands.executeCommand(
-                                "databricks.bundle.showLogs"
-                            );
-                        }
-                    });
-            } else if (e instanceof Error) {
+            if (e instanceof Error) {
```

Comment:
With `allSettled` we are swallowing normal errors right now, we should re-throw them in setTarget/setAuthProvider methods.

But I'm not sure why should we move error handling over there? Seems easier to just have `Promise.all` and catch everything here. Plus we need to add error handling to `ConnectionCommands.selectTarget` too.

Reply:
Removed all settled. I split setting target/auth provider and doing state refresh into 2 separate steps. Now, auth provider and target will still be updated in all models, even if the login fails.